stripe/error.py

```diff
@@ -24,7 +24,8 @@ def __init__(self, message=None, http_body=None, http_status=None,
 
     def __unicode__(self):
         if self.request_id is not None:
-            return u"Request {0}: {1}".format(self.request_id, self._message)
+            msg = self._message or ""
```

Comment:
Okay, so I'm not sure under exactly which conditions we can expect an empty message, but do you thing it would make sense to use a symbolic stand-in instead of an empty string here? e.g. `"<empty message>"` It might make the error more digestible when encountered.